docs/sources/developers/kinds/composable/prometheusdataquery/schema-reference.md

```diff
@@ -15,25 +15,19 @@ title: PrometheusDataQuery kind
+| `refId`      | string  | **Yes**  | *(Inherited from [DataQuery](#dataquery))*<br/>A - Z                                                                                                                                                                                                                                                   |
+| `datasource` |         | No       | *(Inherited from [DataQuery](#dataquery))*<br/>For mixed data sources the selected datasource is on the query level.<br/>For non mixed scenarios this is undefined.<br/>TODO find a better way to do this ^ that's friendly to schema<br/>TODO this shouldn't be unknown but DataSourceRef &#124; null |
+| `editorMode` | string  | No       | Possible values are: `code`, `builder`.                                                                                                                                                                                                                                                                |
+| `exemplar`   | boolean | No       |                                                                                                                                                                                                                                                                                                        |
+| `format`     | string  | No       | Possible values are: `time_series`, `table`, `heatmap`.                                                                                                                                                                                                                                                |
+| `hide`       | boolean | No       | *(Inherited from [DataQuery](#dataquery))*<br/>true if query is disabled (ie should not be returned to the dashboard)                                                                                                                                                                                  |
+| `instant`    | boolean | No       |                                                                                                                                                                                                                                                                                                        |
+| `key`        | string  | No       | *(Inherited from [DataQuery](#dataquery))*<br/>Unique, guid like, string used in explore mode                                                                                                                                                                                                          |
```

Comment:
I was surprised to see "key" here... but then see that is not added in this PR

https://github.com/grafana/grafana/pull/64467